src/Components/Functionalities/IncrementDecrementButton.js

```diff
@@ -0,0 +1,31 @@
+export default function IncrementDecrementButton({ clickHandler, value }) {

```

Comment:
It's so good that you abstracted this file out too.. I really love this. Gonna steal it.. Huhuhahaha.. xD